packages/platform-browser/src/dom/dom_renderer.ts

```diff
@@ -275,8 +275,10 @@ class DefaultDomRenderer2 implements Renderer2 {
 const AT_CHARCODE = (() => '@'.charCodeAt(0))();
 function checkNoSyntheticProp(name: string, nameKind: string) {
   if (name.charCodeAt(0) === AT_CHARCODE) {
-    throw new Error(`Found the synthetic ${nameKind} ${
-        name}. Please include either "BrowserAnimationsModule" or "NoopAnimationsModule" in your application.`);
+    throw new Error(`Unexpected synthetic ${nameKind} ${name} found. Please make sure that:
+  - Either "BrowserAnimationsModule" or "NoopAnimationsModule" are imported in your application.
```

Comment:
I'd propose using backticks here for consistency with the second item:
```suggestion
  - Either \`BrowserAnimationsModule\` or \`NoopAnimationsModule\` are imported in your application.
```